api-tests/fields.test.js

```diff
@@ -7,40 +7,52 @@ const { multiAdapterRunners, setupServer } = require('@keystonejs/test-utils');
 // a while, so we bump up the timeout here.
 jest.setTimeout(60000);
 
-describe('Test CRUD for all fields', () => {
-  const typesLoc = path.resolve('packages/fields/src/types');
-  const testModules = fs
-    .readdirSync(typesLoc)
-    .map(name => `${typesLoc}/${name}/filterTests.js`)
-    .filter(filename => fs.existsSync(filename));
-  testModules.push(path.resolve('packages/fields/tests/idFilterTests.js'));
+describe('Fields', () => {
+  const testModules = globby.sync(`packages/fields/src/types/**/test-fixtures.js`, {
+    absolute: true,
+  });
+  testModules.push(path.resolve('packages/fields/tests/test-fixtures.js'));
 
   multiAdapterRunners().map(({ runner, adapterName }) =>
-    describe(`Adapter: ${adapterName}`, () => {
+    describe(`${adapterName} adapter`, () => {
       testModules.map(require).forEach(mod => {
-        describe(`All the CRUD tests for module: ${mod.name}`, () => {
-          const listName = 'test';
-          const keystoneTestWrapper = (testFn = () => {}) =>
-            runner(
-              () => {
-                const name = `Field tests for ${mod.name} ${cuid()}`;
-                const createLists = keystone => {
-                  // Create a list with all the fields required for testing
-                  const fields = mod.getTestFields();
+        const listName = 'test';
+        const keystoneTestWrapper = (testFn = () => {}) =>
+          runner(
+            () => {
+              const name = `Field tests for ${mod.name} ${cuid()}`;
+              const createLists = keystone => {
+                // Create a list with all the fields required for testing
+                const fields = mod.getTestFields();
 
-                  keystone.createList(listName, { fields });
-                };
-                return setupServer({ name, adapterName, createLists });
-              },
-              async ({ keystone, ...rest }) => {
-                // Populate the database before running the tests
-                await keystone.createItems({ [listName]: mod.initItems() });
+                keystone.createList(listName, { fields });
+              };
+              return setupServer({ name, adapterName, createLists });
+            },
+            async ({ keystone, ...rest }) => {
+              // Populate the database before running the tests
+              await keystone.createItems({ [listName]: mod.initItems() });
 
-                return testFn({ keystone, adapterName, ...rest });
-              }
-            );
+              return testFn({ keystone, listName, adapterName, ...rest });
+            }
+          );
 
-          mod.filterTests(keystoneTestWrapper);
+        describe(`${mod.name} field`, () => {
+          if (mod.crudTests) {
+            describe(`CRUD operations`, () => {
+              mod.crudTests(keystoneTestWrapper);
+            });
+          } else {
+            test.todo('CRUD operations - tests missing');
```

Comment:
Gives nice output when running the tests (without failing the suite):

<img width="555" alt="Screen Shot 2020-03-16 at 12 28 54 am" src="https://user-images.githubusercontent.com/612020/76702342-2888fc00-671d-11ea-8aef-327a5606c96c.png">